src/LandIce/problems/LandIce_Enthalpy.hpp

```diff
@@ -623,6 +623,7 @@ LandIce::Enthalpy::constructEvaluators (PHX::FieldManager<PHAL::AlbanyTraits>& f
       entity = Albany::StateStruct::NodalDistParameter;
       p = stateMgr.registerStateVariable(stateName, dl->node_scalar, elementBlockName, true, &entity, "");
       p->set<std::string>("Parameter Name", stateName);
+      p->set<Teuchos::RCP<Albany::AccessorsMap>>("Accessors", this->getAccessors());
```

Comment:
General comment: I would make the class/file/strings names more verbose: instead of `Accessors`, I would use `ParameterAccessors`, so it's clearer what kind of 'access' we are talking about.

Reply:
Even better `ScalarParameterAccessor`, to clarify that this has nothing to do with distributed parameters.